data/data_test.go

```diff
@@ -0,0 +1,132 @@
+package data
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+)
+
+type TestClient struct {
+	*MeetupClient
+
+	data []byte
+	err  error
+}
+
+func NewClient(data string, err error) TestClient {
+	return TestClient{
+		data: []byte(data),
+		err:  err,
+	}
+}
+
+func (c TestClient) Get(key string) ([]byte, error) {
+	if c.err != nil {
+		return []byte{}, c.err
+	}
+
+	return c.data, c.err
+}
+
+func TestScheduleHasNoEvents(t *testing.T) {
+	s := Schedule{}
+	if len(s.Events) != 0 {
+		t.Fail()
+	}
+}
+
+func TestFetchEventsStoresEvents(t *testing.T) {
+	s := Schedule{key: "Boulder-Gophers"}
+
+	c := NewClient(`[{"id":"id","name":"Event","time":400}]`, nil)
+
+	s.FetchEvents(c)
```

Comment:
You should assert that the error returned her is nil.

Reply:
👍